concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+RonaTas-2011-431-1	1	carpenter	1042	CARPENTER
+RonaTas-2011-431-2	2	cannon, catapult	3202	CANNON
+RonaTas-2011-431-3	3	to fondle, pet, caress	3780	PET
+RonaTas-2011-431-4	4	stream, current	2935	CURRENT (STREAM)
+RonaTas-2011-431-5	5	false, imitation	3069	FALSE
+RonaTas-2011-431-6	6	pied (of an animals coat)		
+RonaTas-2011-431-7	7	apple	1320	APPLE
+RonaTas-2011-431-8	8	barrel (wooden container andstandard quantity)	2548	BARREL
+RonaTas-2011-431-9	9	to nurse, take care of	2472	SUCKLE (BREASTFEED, NURSE)
+RonaTas-2011-431-10	10	to decay, putrify, turn stale	2907	ROT
+RonaTas-2011-431-11	11	small, tiny	1246	SMALL
+RonaTas-2011-431-12	12	to mow	190	MOW
+RonaTas-2011-431-13	13	to decay, putrefy, go bad (of food, water)	2907	ROT
+RonaTas-2011-431-14	14	to harm	44	INJURE
+RonaTas-2011-431-15	15	a type of lasso		
+RonaTas-2011-431-16	16	ditch, canal	2002	DITCH
+RonaTas-2011-431-17	17	barley, Hordeum	932	BARLEY
+RonaTas-2011-431-18	18	barrow	2565	CASTRATED BOAR
+RonaTas-2011-431-19	19	gauntry, gantry (for supporting barrels)		
+RonaTas-2011-431-20	20	charm		
+RonaTas-2011-431-21	21	soggy place, moor, swamp, marsh	1145	SWAMP
+RonaTas-2011-431-22	22	hatchet, axe	677	AXE
+RonaTas-2011-431-23	23	pagan idol, stone or wooden pillar, vertical beam of different wooden instruments, strong, corpulent person	857	STONE
+RonaTas-2011-431-24	24	governor of Croatia, governor of the southern marches		
+RonaTas-2011-431-25	25	to regret, to be sorry	1875	REGRET
+RonaTas-2011-431-26	26	hairy, bushy (of dogs); a big hairy dog (as noun)		
+RonaTas-2011-431-27	27	label, tag		
+RonaTas-2011-431-28	28	cattle, livestock	1153	CATTLE
+RonaTas-2011-431-29	29	velvet		
+RonaTas-2011-431-30	30	to fuck		
```

Comment:
`549 COPULATE`, this can be mapped

Reply:
Ah, I deliberately left this unmapped because I thought from a pragmatic point of view those concepts are quite different. Also I feel like the meaning is somehow narrower? Ah, just checked with [WordNet](http://wordnetweb.princeton.edu/perl/webwn?o2=&o0=1&o8=1&o1=1&o7=&o5=&o9=&o6=&o3=&o4=&s=fuck&i=4&h=01000000#c) and they also list "to fuck" as a **hypernym** of "to copulate"

Reply:
The word never occurred in this form in our data. It depends on your judgment of what you think the authors wanted to denote: a rude, tabooised version of "to copulate" or did they just use it in the way that people also write "shit", to make sure one does not confuse it when they use alternative glosses or that they want to avoid formal-sounding language? If you decide to not map it, you could add the `ignore.tsv` file for the first time, place it in `concepticondata`, add the ID of the entry, the gloss, and a comment, why it is not "to copulate".

Reply:
Alright, will try to add the `ignore.tsv`, sounds like a good exercise.

Reply:
Resolved in 9a25ac393582ec7d459ea2e5ad798b277c1ec186